hail/python/test/hail/table/test_stress.py

```diff
@@ -0,0 +1,23 @@
+import pytest
```

Comment:
do we need this import?

Reply:
this test will also be run with different setup/teardown fixtures from the rest of them. I'd suggest we put it in the mess of test_table.py for now until we can figure out the right thing to do